confidential-data-hub/hub/src/config.rs

```diff
@@ -94,6 +97,13 @@ impl CdhConfig {
             }
         };
 
+        if let std::result::Result::Ok(env) =
+            env::var(CDH_DEFAULT_IMAGE_AUTHENTICATED_REGISTRY_CREDENTIALS)
+        {
+            info!("Read authenticated registry credentials URI from env: {env}");
+            config.image.authenticated_registry_credentials_uri = Some(env);
```

Comment:
this will overwrite the config file. If we want to have the config file to take precedence, we need to move this up. But I think the usual behaviour is ENV > config-file, so we might just want to change the remark in example.config.toml